test/test_transfer.cpp

```diff
@@ -104,10 +103,10 @@ struct test_storage : default_storage
 			return 0;
 		}
 
-		for (int i = 0; i < num_bufs; ++i)
+		for (int i = 0; i < int(bufs.size()); ++i)
```

Comment:
I think you could just do: for (auto const& b : bufs) m_written += int(b.iov_len);
here